eng/pipelines/templates/stages/archetype-autorest-preview.yml

```diff
@@ -0,0 +1,336 @@
+parameters:
```

Comment:
Is the goal to import these templates directly from the tools repo into the autorest pipelines?

Reply:
That's correct.   The autorest pipelines include 2 repo resources, azure-sdk-tools and azure-sdk-for-lang.  They extend the template like 

``` yaml
extends:
  template: eng/pipelines/templates/stages/archetype-autorest-preview.yml@azure-sdk-tools
  parameters:
    ...
```

Reply:
It's not needed by any of the repos that sync eng/common, and it's easier to pin the tools repo ref than set up another common folder sync pipeline